spec/models/validations/sales/household_validations_spec.rb

```diff
@@ -377,21 +337,8 @@
   end
 
   describe "#validate_buyer1_previous_tenure" do
-    let(:record) { build(:sales_log) }
-
-    let(:now) { Time.zone.local(2024, 4, 4) }
-
-    before do
-      Timecop.freeze(now)
-      Singleton.__init__(FormHandler)
-      record.ownershipsch = 2
-      record.saledate = now
-    end
-
-    after do
-      Timecop.return
-      Singleton.__init__(FormHandler)
-    end
+    let(:saledate) { Time.zone.local(2024, 4, 4) }
```

Comment:
Same for this - does it work if this is now?

Reply:
Yeah, I guess I wanted this to be explicit that it's on/after 2024 cause there are also tests for 2023 specific behaviour, but I guess everything from now on will be on or after 2024, so will remove